test/timer.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { GameInfoType, SlippiGame } from "../src";
+import { frameToGameTimer } from "../src/utils/gameTimer";
+
+describe("when calculating the in-game timer", () => {
+  it("should calculate the timer correctly when increasing", () => {
+    const timer_increasing = new SlippiGame("./slp/timer_increasing.slp");
+    const timer_increasing_gameinfo = timer_increasing.getSettings()?.gameInfo as GameInfoType;
+    expect(frameToGameTimer(timer_increasing.getMetadata()?.lastFrame as number, timer_increasing_gameinfo)).toBe(
+      "00:33.57",
+    );
+  });
+
+  it("should calcualte the timer correctly when the limit is hit", () => {
+    const timer_limit = new SlippiGame("./slp/timer_limit.slp");
+    const timer_limit_gameinfo = timer_limit.getSettings()?.gameInfo as GameInfoType;
+    expect(frameToGameTimer(timer_limit.getMetadata()?.lastFrame as number, timer_limit_gameinfo)).toBe("00:00.00");
+  });
+
+  it("should calculate the timer correctly when decreasing", () => {
+    const timer_decreasing = new SlippiGame("./slp/timer_decreasing.slp");
+    const timer_decreasing_gameinfo = timer_decreasing.getSettings()?.gameInfo as GameInfoType;
+    expect(frameToGameTimer(timer_decreasing.getMetadata()?.lastFrame as number, timer_decreasing_gameinfo)).toBe(
+      "01:51.76",
+    );
+  });
```

Comment:
You don't need to parse an entire`SLP` file just to test the timer logic. You can simply pass a `lastFrame` number and duration to the conversion function.